.env.example

```diff
@@ -74,16 +58,48 @@ REDIS_TLS_CA_FILE=./certs/redisCA.crt
 REDIS_TLS_REJECT_UNAUTHORIZED='false'
 RETHINKDB_URL='rethinkdb://localhost:28015/actionDevelopment'
 RETHINKDB_SSL='false'
-# SEGMENT_WRITE_KEY='key_SEGMENT_WRITE_KEY'
-SENTRY_AUTH_TOKEN='key_SENTRY_AUTH_TOKEN'
-# SENTRY_DSN='key_SENTRY_DSN'
+
+# INTEGRATIONS
+ATLASSIAN_CLIENT_ID='key_ATLASSIAN_CLIENT_ID'
+ATLASSIAN_CLIENT_SECRET='key_ATLASSIAN_CLIENT_SECRET'
+GITHUB_CLIENT_ID='key_GITHUB_CLIENT_ID'
+GITHUB_CLIENT_SECRET='key_GITHUB_CLIENT_SECRET'
+GITHUB_WEBHOOK_SECRET='key_GITHUB_WEBHOOK_SECRET'
+GITLAB_CLIENT_ID='key_GITLAB_CLIENT_ID'
+GITLAB_CLIENT_SECRET='key_GITLAB_CLIENT_SECRET'
+GOOGLE_CLOUD_CLIENT_EMAIL='key_GOOGLE_CLOUD_CLIENT_EMAIL'
+GOOGLE_CLOUD_PRIVATE_KEY='key_GOOGLE_CLOUD_PRIVATE_KEY'
+GOOGLE_CLOUD_PRIVATE_KEY_ID='key_GOOGLE_CLOUD_PRIVATE_KEY_ID'
 # Cluster node number 0 - 1023
-SERVER_ID='1'
 SLACK_CLIENT_ID='key_SLACK_CLIENT_ID'
 SLACK_CLIENT_SECRET='key_SLACK_CLIENT_SECRET'
-SOCKET_PORT='3001'
-# https://stripe.com/docs/keys
+
+# MAIL
+# MAIL GLOBALS. PROVIDER: mailgun | google | debug | smtp
+MAIL_PROVIDER='debug'
+MAIL_FROM='key_MAIL_FROM'
+MAILGUN_API_KEY='key_MAILGUN_API_KEY'
+MAILGUN_DOMAIN='key_MAILGUN_DOMAIN'
+MAIL_GOOGLE_USER='key_MAIL_GOOGLE_USER'
+MAIL_GOOGLE_PASS='key_MAIL_GOOGLE_PASS'
+MAIL_SMTP_HOST='key_MAIL_SMTP_HOST'
+MAIL_SMTP_PORT=587
+MAIL_SMTP_USER='key_MAIL_SMTP_USER'
+MAIL_SMTP_PASSWORD='key_MAIL_SMTP_PASSWORD'
+MAIL_SMTP_USE_TLS='1' # set to '0' for false
+MAIL_SMTP_CIPHERS='HIGH:MEDIUM:!aNULL:!eNULL:@STRENGTH:!DH:!kEDH'
+
+# STRIPE KEYS https://stripe.com/docs/keys
 STRIPE_SECRET_KEY='sk_test_4eC39HqLyjWDarjtT1zdp7dc'
 STRIPE_PUBLISHABLE_KEY='pk_test_TYooMQauvdEDq54NiTphI7jx'
 STRIPE_WEBHOOK_SECRET='sk_test_4eC39HqLyjWDarjtT1zdp7dc'
+
+# OTHER
+CDN_BASE_URL=''
+CI='true'
+FILE_STORE_PROVIDER='s3'
+HOST='localhost'
+MIN_LOG_DURATION='200'
```

Comment:
+1 I cannot find a reference for `MIN_LOG_DURATION`, we should remove it

Reply:
Done ✅